src/components/DittoText.tsx

```diff
@@ -4,9 +4,9 @@ import { useProjectId } from "../lib/utils";
 import { DittoTextProps } from "./Ditto";
 
 export const DittoText = (props: DittoTextProps) => {
-  const { textId, children } = props;
+  const { textId, children, variables = {}, count } = props;
   const projectId = useProjectId(props);
-  const text = useDittoSingleText({ projectId, textId });
+  const text = useDittoSingleText({ projectId, textId, variables, count });
```

Comment:
I think instead of having a default argument for variables, we should call this like so:
```suggestion
  const text = useDittoSingleText({ projectId, textId, variables: variables || {}, count });
```
^^ handles the case where `variables` is `null`, whereas the default argument does not (and I could easily see `null` being passed in from user land)

Reply:
👍